### ISSUE TITLE
Angular Universal (SSR) support

### ISSUE THREAD

**Issue by lehoffma** [amcharts/amcharts4#1295]
Hey,
I'm trying to run an angular universal app with amcharts, but my app just crashes with multiple errors like this one: 

```
ERROR TypeError: css_element_queries_src_ResizeSensor__WEBPACK_IMPORTED_MODULE_8___default.a is not a constructor  
    at new SVGContainer (<path/to/my/server/file>)
```

The stack trace points to [this line of code](https://github.com/amcharts/amcharts4/blob/bc3a4c53184ac889beafd0830c3107a395ee742f/src/.internal/core/rendering/SVGContainer.ts#L132). It seems that `ResizeSensor` is just not available in Angular Universal builds, but I couldn't find more information about it yet. 

I'm using the newest `amcharts4` version (v4.4.4).

**Comment by martynasma** [comment]
Currently we do not have specific support for SSR.

We are making a note to investigate it.

Will keep you posted.

**Comment by lehoffma** [comment]
Are there any updates? I'm just asking because you closed the issue and I didn't see anything relevant in the changelogs.

**Comment by martynasma** [comment]
We'll provide a solution to run amCharts 4 with SSR with the next version.

Will let you know details when the new version is out. Possibly by the end of this week.

**Comment by martynasma** [comment]
Fixed in [4.10.0](https://github.com/amcharts/amcharts4/releases/tag/4.10.0).

## [4.10.0] - 2020-09-01

### Added
- New `LineSeries` property: `smoothing` (`"bezier"` (default), `"monotoneX"`, and `"monotoneY"`) - indicates algorithm to use for smoothing the lines. Monotone algos are better suited for irregularly-spaced data. [More info](https://www.amcharts.com/docs/v4/chart-types/xy-chart/#Alternate_smoothing_algorithm).
- `ForcedDirectedTree`: new property `zoomable` (default: `false`). If set to `true`, the chart will be zoomable/draggable using mouse, touch, or API. [More information](https://www.amcharts.com/docs/v4/chart-types/force-directed/#Zooming).
- `ForcedDirectedTree`: zoom related properties added. `mouseWheelBehavior` (`"zoom"`, `"none"`) - controls what happens when mouse wheel is used over chart; `zoomStep` (default: `2`) - step to increase zoom level my each zoom in; `zoomOutButton` - an instance of button which is shown when chart is zoomed in.
- `ForcedDirectedTree`: new zoom-related methods: `zoomToPoint(point, level, center)` and `zoomToDataItem(dataItem, level, center)`;
- New global function: `am4core.createDeferred(callback, scope, ...parameters)`. Can be used to create multiple charts in a truly safe daisy-chain (next chart starts to be instantiated only when previous triggers a `ready` event). Returns a `Promise` with chart instance. [More information](https://www.amcharts.com/docs/v4/concepts/performance/#Deferred_daisy_chained_instantiation);
- New global option: `am4core.options.deferredDelay` (default `100`). Specifies time-gap in milliseconds between deferred chart instantiations.

### Changed
- Added date ordinal translations for Chinese locales.

### Fixed
- Accessibility: `readerTitle` was not being properly populated with data on `Label`.
- Accessibility: `readerDescription` was not being populated with data.
- Now works with Angular Universal SSR. Please refer to our [Angular doc](https://www.amcharts.com/docs/v4/getting-started/integrations/using-angular2/) for instructions on how to structure your app for SSR to work properly.
- When `MapChart` was panned with inertia and a polygon with `zoomToObject()` hit event was clicked when the map was still moving, the map was zooming into a wrong position.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
